tests/staking/tier-threshold.test.ts

```diff
@@ -21,6 +21,17 @@ given('astar')('Number of slots adjusted based on price', async ({ networks: { a
 
   const palletVersion = (await astar.api.query.dappStaking.palletVersion<u16>()).toNumber()
 
+  const inflationConfig = (await astar.api.query.inflation.activeInflationConfig()).toJSON() as Record<string, any>
+  const noRewardsConfig = {
+    ...inflationConfig,
+    collatorRewardPerBlock: 0n,
+    treasuryRewardPerBlock: 0n,
+    dappRewardPoolPerEra: 0n,
+    baseStakerRewardPoolPerEra: 0n,
+    adjustableStakerRewardPoolPerEra: 0n,
+    bonusRewardPoolPerPeriod: 0n,
+  }
```

Comment:
@ipapandinas what's the point of this modification?

Reply:
To make snapshots more deterministic. Advancing to the next era emits rewards, which affects total issuance and the tier thresholds. With potential future decayed emissions, this would vary more and make snapshots unpredictable. I am controlling the inputs to isolate and test the tier threshold calculations directly. Another option is to make the assertions more lenient for tier thresholds.